universal-application-tool-0.0.1/app/controllers/HomeController.java

```diff
@@ -82,6 +86,18 @@ public Result loginForm(Http.Request request, Optional<String> message)
     return ok(loginForm.render(request, messagesApi.preferred(request), message));
   }
 
+  public Result robotsTxt() {
+    if (environment.isProd()) {
+      return ok("User-agent: *\n" +
+              "Allow: /$\n" +
+              "Allow: /loginForm$\n" +
```

Comment:
do we need to end with $ here?

Reply:
> do we need to use https://developers.google.com/search/docs/advanced/crawling/block-indexing to block indexing?

After doing to looking it sounds like using meta robots in tandem with robots.txt is often recommended these days. I'll look into adding that as well.

> do we need to end with $ here?

I was looking to specifically target the site index and the loginForm page. The $ will make sure to match to the end of the string. My understanding is that without the $ it could end up matching loginForm, loginForm2, loginForm/nested/path.

Reply:
cool feel free to add this and do the meta stuff in another PR